### DISCUSSION TITLE
Lost in the concept

### DISCUSSION THREAD

**Discussion by lisachev** [immich-app/immich#5484] [Q&A] [answered]
Hi
I'm a returning user to Immich. With the rapid development it has changed quite a lot in the past few months! I take my hat off to @alextran1502!
One thing I'm still struggling with is the 'Backup' terminology. Can anyone give me a definition? In particular, I can't find a clear answer to these questions:
1. Does the app copy selected albums media in full resolution/quality?
2. Will the server side backup contain media that has been removed by the user on the device outside of Android/iOS app? Acting like a backup vs sync.
3. What are the transcoding settings on the server side for if the backup is in full resolution?
4. For iOS would the following be true: once all images are taken off iCloud and uploaded to Immich, Photos app should probably be replaced by the Immich app in a day-to-day workflow

**Comment by bo0tzz** [comment, marked as answer]
1. Yes
2. Yes
3. Compatibility with different clients
4. Yes, with the caveat that Immich is still under heavy development and you should not use it as your only backup solution.
